app/controllers/org_admin/template_options_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class OrgAdmin::TemplateOptionsController < ApplicationController
```

Comment:
I like the refactor of this out of the templates controller into it's own.

Semantically, I dont think this shouldn't be under the org-admin module.
The action is accessed when regular users are trying to create a plan.

Reply:
Yes, I agree. 

I kept it there because that's where it was originally. But I've just pushed out a new commit to move it into the main namespace.